### PR TITLE
docs(operators): add documentation for pairwise

### PR DESCRIPTION
**Pull request by sumitarora**
Closes: https://github.com/ReactiveX/rxjs-docs/issues/68

### CODE REVIEW

**Comment by JWO719** [comment]
For me it looks good. Is it ok to give an approval, or should this be made by "core contributors"?

**Comment by sumitarora** [comment]
@JWO719 You can also approve it.

**Comment by ashwin-sureshkumar** [comment]
@sumitarora - can you rebase, also can you please fix single quotes.

File: src/operator-docs/combination/pairwise.ts [hunk 1]
```diff
@@ -1,6 +1,51 @@
-import { OperatorDoc } from '../operator.model';
+import { OperatorDoc } from "../operator.model";
 
 export const pairwise: OperatorDoc = {
-  'name': 'pairwise',
-  'operatorType': 'combination'
+  name: "pairwise",
```
Comment (ashwin-sureshkumar) [inline src/operator-docs/combination/pairwise.ts]: please fix the quotes.

**Comment by sumitarora** [comment]
@ashwin-sureshkumar done

**Pull request merged**